XIPs/xip-0-purpose-process.md

```diff
@@ -38,13 +38,21 @@ An XIP must meet certain minimum criteria. It must be a clear and complete descr
 
 ## XIP Workflow
 
-## Getting started
+### Get started with an idea
 
 Before advancing your idea through the XIP process into the Draft stage, it’s crucial to gather feedback from the community. This will help you vet your idea and assess its originality, potentially saving you time and effort. Engaging with the XMTP community will also help you gauge overall interest in your idea.
 
-Prototyping your proposal may also help to de-risk it and provide a reference implementation to anchor discussions around.
+**To get started with your idea**:
 
-To get started, you can **submit your idea as a new topic in the [XIP Ideas category in XMTP's GitHub Discussions](https://github.com/orgs/xmtp/discussions/categories/xip-ideas)**.
+1. Submit your idea as a new topic in the [Ideas & Improvements category](https://community.xmtp.org/c/development/ideas/54) in the XMTP Community Forums.
+
+1. If you've built a prototype for your idea, provide access to it in your topic. A prototype can help de-risk your idea and give people something to anchor discussions around.
+
+1. Be sure to include the date by which you want people to provide feedback. A good standard is giving people ~2 weeks, including 2 weekends.
+
+1. Share your idea with the community to get feedback. For example, beyond posting to the XMTP Community Forums, you might want to share a link to your idea topic with your networks on other platforms.
+
+1. After you feel your idea has received enough feedback, you can create an XIP Draft for it, as described in the next section. A good standard for "enough feedback" is at least one comment from an XMTP community member and one from an *XMTP Core Developer* (currently the XMTP Labs team).
```

Comment:
Should this be phrased as something more along the lines of "After feedback date has passed and there is at least 1 internal and 1 external review" -- just thinking we may want to make this step more concrete

Reply:
Because this documentation is meant for the general public - I was thinking that they would not understand what we mean by 1 internal and 1 external review. <3 For the general public, I believe internal translates to "XMTP Core Developers" - who are designated as members of the XMTP Labs team in Discourse, and external translates to an XMTP community member. Does this make sense? =)

Reply:
Yes! Absolutely. Those are much better choices given the audience - thanks for the clarification!